django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -195,4 +195,12 @@ Requires core.js, SelectBox.js and addevent.js.
         }
     };
 
+    addEvent(window, "load", function(e) {
```

Comment:
I could be missing something obvious, but any reason to prefer `addEvent` over jQuery style: `$(window).load(function() {`